libs/core/src/components/sage-tabpanel/sage-tabpanel.tsx

```diff
@@ -0,0 +1,48 @@
+import { Component, Element, Host, h, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'sage-tabpanel',
+  styleUrl: 'sage-tabpanel.scss',
+  shadow: false,
+})
+export class SageTabpanel {
+  @Element() el: HTMLElement;
+
+  /**
+   * Sets the related tab name, this name must match a `sage-tab`'s tab property
+  */
+  @Prop() tab!: string;
+  
+  /**
+   * Keeps track of the parentComponent unique id, this property is passed by parent component
+  */
+  /** @internal */
+  @Prop({mutable: true}) parentComponent: string;
+
+  /**
+   * Keeps track of if the expected tab variant, this property is passed by parent component
+  */
+   /** @internal */
+   @Prop() variant: string;
+
+  /**
+   * Keeps track of the tabpanel selected state, this property is passed by parent component 
+  */
+  /** @internal */
+  @Prop({mutable: true}) selected = false;
+
+  render() {
+    return (
+      <Host slot="tabpanels">
+        <div 
+          role="tabpanel"
```

Comment:
Since we're aligning to the W3C implementation, should each panel also be set to `tabindex="0"`?

Reply:
Good callout, I'll update!